airflow/hooks/postgres_hook.py

```diff
@@ -13,9 +15,14 @@ class PostgresHook(DbApiHook):
 
     def get_conn(self):
         conn = self.get_connection(self.postgres_conn_id)
-        return psycopg2.connect(
+        conn_args = dict(
             host=conn.host,
             user=conn.login,
             password=conn.password,
             dbname=conn.schema,
             port=conn.port)
+        # check for ssl parameters in conn.extra
+        for arg_name, arg_val in conn.extra_dejson.iteritems():
```

Comment:
`iteritems` was taken away in py3, you can just use `items` in place of it. It's a fn in py2 and a generator in py3 but will work identically for a small dict.